config/default.json

```diff
@@ -14,7 +14,20 @@
   "validations": {
     "email": "^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$"
   },
+  "auth": {
+    "excludeAuthPaths": [
+      {
+        "method": "POST",
+        "endpoint": "/api/identity"
+      }
+    ]
+  },
+  "uniqueness": {
+    "devices": ["sms", "email", "apn", "gcm", "phone"],
+    "channels": true
+  },
   "test": {
-    "fixtures_folder": "../fixtures/"
+    "fixtures_folder": "/fixtures/",
+    "resources_folder": "/cucumber/test_files/"
```

Comment:
Maybe we can separate user config from internal config. This `test` node refers to app config and should be added elsewhere.